src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/AndroidSdkResolver.cs

```diff
@@ -117,8 +117,16 @@ static string RunPathsTargets (string target)
 		}
 
 		static int? maxInstalled;
+		static object lockObject = new object ();
```

Comment:
Should this have a name which better describes the "scope" that it's protecting?  Perhaps `maxInstalledLock`?